faker/providers/date_time/__init__.py

```diff
@@ -1426,16 +1426,16 @@ def _rand_seconds(self, start_datetime: int, end_datetime: int) -> float:
         ),
         Country(
             timezones=[
-                "Europe/Kiev",
-                "Europe/Uzhgorod",
-                "Europe/Zaporozhye",
+                "Europe/Kyiv",
+                "Europe/Uzhhorod",
+                "Europe/Zaporizhzhia",
```

Comment:
Similarly, this should still be `"Europe/Zaporozhye"`

Reply:
I agree, I initially misunderstood the meanings.

Reply:
This should be changed back to `"Europe/Zaporozhye"`